### ISSUE TITLE
k8s ephemeral volumes ETL warning

### ISSUE THREAD

**Issue by Omar-Bishtawi** [kubecost/cost-analyzer-helm-chart#1889]
**Describe the bug**  
this warning message keeps appearing for k8s ephemeral volumes 
```bash
2022-12-28T11:47:05.31116268Z WRN Asset ETL: disk 'pvc-654758a2-421c-4dbe-8d95-5b525273ffa8' start outside window: 0001-01-01T00:00:00 not in [2022-12-28T11:00:00, 2022-12-28T12:00:00]
2022-12-28T11:47:05.311175779Z WRN Asset ETL: disk 'pvc-654758a2-421c-4dbe-8d95-5b525273ffa8' end outside window: 0001-01-01T00:00:00 not in [2022-12-28T11:00:00, 2022-12-28T12:00:00]
```
and the number of logs keeps increasing as you see in the following picture
<img width="897" alt="image" src="https://user-images.githubusercontent.com/91823055/209810419-41d13eae-e4fe-459a-9b77-5e9af9e875a0.png">
because once an ephemeral PVC is introduced to the system even if it was deleted kubecost asset ETL keep trying to validate the asset. 

in the following picture, the blue bar represents when a single PVC was created and deleted while the rest red bars are for the above two log lines.
<img width="1663" alt="image" src="https://user-images.githubusercontent.com/91823055/209810712-bb57da73-05d1-4561-bf34-d956140ff589.png">

one note I don't know if it's relevant. in the ETL assets section from the kubecost dashboard, all ETL files are showing that they have valid ETL data stored

**To Reproduce**  
create a pod and attach an ephemeral volume attached to it. 
```yaml
volumes:
      - name: sample-volume
        ephemeral:
          volumeClaimTemplate:
            spec:
              accessModes:
              - ReadWriteOnce
              resources:
                requests:
                  storage: 4Gi
              storageClassName: ebs-csi-gp3-baseline-sc-backend
```

**Expected behavior**  
to deal with ephemeral volumes the same way kubecost deal with normal volumes. and not show this warning message.

**Comment by teevans** [comment]
Hey @Omar-Bishtawi - Thanks for opening this - Most of our team is out of office this week, But we'll see what we can do getting an adjustment for this into 1.100.

CC: @nikovacevic / @Adam-Stack-PM

**Comment by Omar-Bishtawi** [comment]
@teevans thank you for your help. 
after I updated from V1.98 to V1.99 this error didn't happen again. I checked the release notes, there were a couple of changes related to PVC. it seems These updates resolved this issue. if you don't have any further notes regarding this I think there is no need to leave this ticket open and it can be closed.

**Comment by teevans** [comment]
Awesome - Thanks @Omar-Bishtawi! I'll go ahead and close it out.
